blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -81,36 +95,50 @@ public int getIntegerProperty(Enum<? extends ConfigDefaults> name) {
         return getIntegerProperty(name.toString());
     }
```

Comment:
These methods are responsible for converting values from strings to whatever datatype is desired.
By separating them into their own methods, we can test them without having to manually set system properties.
The ones you mention,( `intFromString`, `longFromString`, and `floatFromString`) are trivially implemented, because they just call the system parse methods.
Testing these methods ensures they function correctly, although since they're trivial enough that that's easy to tell just from looking at them.
However, the tests also serve as a kind of documentation of how the conversion works now.

We may very well decide to change the number format in the future, and these methods would be useful for that.
As an example, the way it works now, the conversion can't handle certain corner-cases, like leading or trailing whitespace.
If for some reason a configuration value had whitespace in it, an exception would be thrown.
To avoid that, we could change how the values were parsed.
Having the conversion in a separate method makes it easy to know where to make the change, and also makes it easy to test the change.